container/go/pkg/oci/reader_test.go

```diff
@@ -0,0 +1,108 @@
+package oci
+
+import (
+	"testing"
+
+	v1 "github.com/google/go-containerregistry/pkg/v1"
+	"github.com/google/go-containerregistry/pkg/v1/types"
+	"github.com/google/go-containerregistry/pkg/v1/validate"
+)
+
+var readertests = []struct {
+	manifestDigest v1.Hash
+	configDigest   v1.Hash
+	layerHashes    []v1.Hash
+	mediaType      types.MediaType
+	testPath       string
```

Comment:
Add a `name` field to give each test case a short descriptive name & print that when you call t.Run